packages/radio-group/src/vaadin-radio-group.js

```diff
@@ -249,6 +249,23 @@ class RadioGroup extends FieldMixin(
     }
   }
 
+  /**
+   * Override an observer from `FieldMixin`.
+   *
+   * @param {boolean} invalid
+   * @protected
+   * @override
+   */
+  _invalidChanged(invalid) {
+    super._invalidChanged(invalid);
+
+    if (invalid) {
+      this.setAttribute('aria-invalid', 'true');
+    } else {
+      this.removeAttribute('aria-invalid');
```

Comment:
I would suggest we also add a unit test as the `false` branch is currently not tested at all.

Reply:
And how about we use `DelegateStateMixin` for the purpose of delegating `invalid`?

Reply:
Technically, we could use the mixin but it feels a bit weird because the attribute is not supposed to be delegated to other element, but set on the host.

Reply:
Ah, indeed. 👍